### ISSUE TITLE
Changing the root font ? Really ?

### ISSUE THREAD

**Issue by fredericfalliere** [zachacole/Simple-Grid#29]
Your mission statement is to be lightweight. I don't understand why simple-grid would be responsible for defining the font, color in `html, body` and `*`.

https://github.com/zachacole/Simple-Grid/blob/master/simple-grid.css#L24

**Comment by sapientorius** [comment]
I agree. Just removed it. As well as the other changes to h1, p etc. (AND import lato via google fonts)

Would prefer, if the project just provides the grid and nothing else.

**Comment by mmontag** [comment]
<img src="https://c.tenor.com/FfHozmky4I0AAAAd/simpsons-leaving.gif"/>
